toolprovider/mise/bootstrap.go

```diff
@@ -59,7 +60,11 @@ func installReleaseBinary(version string, checksums map[string]string, targetDir
 // and returns the path to the downloaded file.
 func downloadAndVerify(url, expectedChecksum string) (string, error) {
 	// Get the tarball, check status
-	resp, err := retryablehttp.Get(url)
+	logger := log.NewLogger(log.GetGlobalLoggerOpts())
+	client := retryablehttp.NewClient()
+	client.Logger = &log.HTTPLogAdaptor{Logger: logger}
```

Comment:
This logic is wrapped in [retryhttp. NewClient](https://github.com/bitrise-io/go-utils/blob/master/retryhttp/retryhttp.go#L9), couldn't we use it?